src/MLPlots.jl

```diff
@@ -2,12 +2,13 @@ module MLPlots
 
 using Reexport
 @reexport using Plots
-using LearnBase
+using Requires
 
 export
     corrplot
 
 include("recipes.jl")
-include("loss.jl")
+@require LearnBase include("loss.jl")
+@require ValueHistories include("valuehistories.jl")
```

Comment:
actually I am thinking of going one step further and creating subfolder for each package that has a typical module-like file which handles the conditional imports and exports and optional includes

```Julia
@require LearnBase include("LearnBase/LearnBase.jl")
```

Packages with little code could directly put it into that file, while bigger packages could have split up the implementations across other files in their folders (similar to what modules typical do). Since we will probably end up with a lot of packages I think this would be beneficial in the long run.